reportssql/treatment_report_functions.sql

```diff
@@ -113,7 +113,8 @@ CREATE FUNCTION TREATMENT_Indicator3(
     p_startAge INT(11),
     p_endAge INT (11),
     p_includeEndAge TINYINT(1),
-    p_gender VARCHAR(1)) RETURNS INT(11)
+    p_gender VARCHAR(1),
+    p_isBreastfeeding TINYINT(1)) RETURNS INT(11)
```

Comment:
@lauravignoli when this flag is false, the query should count all patients (irrespective of whether or not they are breastfeeding); and true => count only breastfeeding patients. I'd suggest we rename the variable to something like `p_includeOnlyBreastfeeding` for more clarity

Reply:
Done